todo-app/src/Task.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import AppTask from './AppTask';
```

Comment:
What is your `AppTask` component? You are importing it here, but never use it. If it's not used anymore, it should be removed, or at least not imported.